### PR TITLE
Scaffold video archive

### PR DESCRIPTION
**Pull request by neener**

### CODE REVIEW

File: archive-video-award.php [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+  while (have_posts()) {
+    the_post();
+    $video_year = get_post_meta($post->ID, '_igv_video_award_video_year', true);
+    $artists = get_post_meta($post->ID, '_igv_video_award_artists', true);
+    $bio = get_post_meta($post->ID, '_igv_video_award_bio', true);
+    $minutes = get_post_meta($post->ID, '_igv_video_award_minutes', true);
+    $seconds = get_post_meta($post->ID, '_igv_video_award_seconds', true);
+
+    if ($wp_query->current_post == 0) {
```
Comment (mikeyrayvon) [inline archive-video-award.php]: some of these meta values are only used for the first video. you should request those within the conditional.

File: archive-video-award.php [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        
+?>
+      <div class="information-overlay item item-m-4">
```
Comment (mikeyrayvon) [inline archive-video-award.php]: is `item` meant to be `grid-item`?

File: archive-video-award.php [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+  
+<?php
+} else { //grid post
+?> 
+
+      <div class="grid-item item-s-4 background-cover font-uppercase" style="background-image: url(<?php echo the_post_thumbnail_url('full'); ?>)">
+        <h2><?php the_title(); ?></h2>
+        <div><?php echo $artists; ?></div>
+        <div><?php echo get_the_date('F Y'); ?></div>
+        <div class="font-uppercase">information</div>
```
Comment (mikeyrayvon) [inline archive-video-award.php]: there should only be an `information` button for the first post

**Review by mikeyrayvon — CHANGES_REQUESTED** [review]

File: archive-video-award.php [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+  while (have_posts()) {
+    the_post();
+    
+    $artists = get_post_meta($post->ID, '_igv_video_award_artists', true);
+    
+    if ($wp_query->current_post == 0) {
+    $video_year = get_post_meta($post->ID, '_igv_video_award_video_year', true);
+    $bio = get_post_meta($post->ID, '_igv_video_award_bio', true);
+    $minutes = get_post_meta($post->ID, '_igv_video_award_minutes', true);
+    $seconds = get_post_meta($post->ID, '_igv_video_award_seconds', true);
```
Comment (mixcas) [inline archive-video-award.php]: All these should be indented 1 level inside.

**Pull request merged**